### ISSUE TITLE
error: 'video_tick' marked 'override' but does not override any member functions

### ISSUE THREAD

**Issue by yurivict** [Xaymar/obs-StreamFX#146]
Compiler: clang-8
OS: FreeBSD 12.1
Version: 0.8.0.a4
OBS Studio version: 25.0.4
Full log: http://beefy5.nyi.freebsd.org/data/121i386-default/530943/logs/obs-streamfx-0.8.0.a4.log

```
/wrkdirs/usr/ports/multimedia/obs-streamfx/work/obs-StreamFX-0.8.0a4/source/filters/filter-shader.hpp:43:16: error: 'video_tick' marked 'override' but does not override any member functions
                virtual void video_tick(float_t sec_since_last) override;
                             ^
/wrkdirs/usr/ports/multimedia/obs-streamfx/work/obs-StreamFX-0.8.0a4/source/filters/filter-shader.hpp:43:16: warning: 'filter::shader::shader_instance::video_tick' hides overloaded virtual function [-Woverloaded-virtual]
/wrkdirs/usr/ports/multimedia/obs-streamfx/work/obs-StreamFX-0.8.0a4/source/obs/obs-source-factory.hpp:547:16: note: hidden overloaded virtual function 'obs::source_instance::video_tick' declared here: type mismatch at 1st parameter ('float' vs 'float_t' (aka 'long double'))
                virtual void video_tick(float seconds) {}
                             ^
```

**Comment by Xaymar** [comment]
This is probably a bigger issue. Lots of files mix and match old style and new fixed size style values. Will fix it shortly.

**Comment by yurivict** [comment]
```0.8.0.b1``` still fails with the same error: http://beefy10.nyi.freebsd.org/data/113i386-default/531468/logs/obs-streamfx-0.8.0.b1.log

**Comment by Xaymar** [comment]
@yurivict Please test with the fixes in #165 applied.

**Comment by yurivict** [comment]
I've backported this patch to the FreeBSD port, and will observe it for a few days to see if any builds would fail.

**Comment by Xaymar** [comment]
This should be fully fixed by #165, as evidenced by clang-10 building just fine in CI.

**Comment by yurivict** [comment]
I didn't get any more failure notices.
